addon/components/markdown-to-html.js

```diff
@@ -25,7 +25,7 @@ const ShowdownComponent = Ember.Component.extend({
       }
     }
 
-    return new Handlebars.SafeString(this.converter.makeHtml(get(this, 'markdown')));
+    return Ember.String.htmlSafeg(this.converter.makeHtml(get(this, 'markdown')));
```

Comment:
This will not work in anything less than stable so we need to support both, i.e., check before invoking the method.